templates/automotive/index.html

```diff
@@ -0,0 +1,206 @@
+      </div>
+    </div>
+  </div>
+</section>
+  
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>Build innovative applications for connected vehicles</h2>
+      <p>We offer the technology primitives that new automotive applications can be built quickly.</p>
+    </div>
+  </div>
+  <div class="row">
+    <ul class="p-matrix p-matrix--two-col u-clearfix">
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">Digital Car Cockpit</h3>
+          <p class="p-matrix__desc">As customers expectations for digital experiences evolve, car manufacturers need to rethink cockpit user experience to deliver an intuitive UX for infotainment, navigation and enhanced passenger safety. Canonical has extensive UX and HMI experience, thanks to Ubuntu Desktop. This experience is encapsulated in our display solutions; Wayland and Mir.</p>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">ADAS</h3>
+          <p class="p-matrix__desc">Enhanced passenger safety is a key benefit of smart cars. Complex sensors need to be integrated into the car to map traffic situations dynamically. Furthermore, advanced communication protocols need to be implemented to transmit the collected data. Ubuntu is open source, with an active developer community. Choosing Ubuntu lowers the technical challenge of integrating sophisticated new technologies.</p>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">Connected ECUs</h3>
+          <p class="p-matrix__desc">We provide the foundations for smart connected ECUs in a mixed-criticality architecture. Our full abstraction of the hardware layer in embedded systems unlocks new possibilities like time-sensitive networking in automotive systems and ECU virtualisation. We collaborate with experienced partners for compliance with safety standards.</p>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">V2X</h3>
+          <p class="p-matrix__desc">V2X requires the support for a variety of communication stacks and protocols, from the vehicle to other vehicles, infrastructure or roadway users. Our approach to V2X is software-defined networking. We build on infrastructure abstraction and automation, leveraging our edge version of Kubernetes, Snap application containers and cloud orchestration tools like Juju. Enabling hardware standardisation and faster development lead time for V2X solutions.</p>
+        </div>
+      </li>
+    </ul>
+  </div>
+</section>
+    
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <div class="p-strip is-shallow u-no-padding--top">
```

Comment:
I don't like the use of a strip inside a strip. If you need more margin please use `u-sv3` on the heading.

Reply:
Fixed